torch/_inductor/codecache.py

```diff
@@ -1247,6 +1247,8 @@ def optimization_flags() -> str:
         base_flags += " -fno-unsafe-math-optimizations"
     if not config.cpp.enable_floating_point_contract_flag:
         base_flags += " -ffp-contract=off"
+    if not config.cpp.enable_tree_loop_vec_opt_flag:
+        base_flags += " -fno-tree-loop-vectorize"
```

Comment:
Please add the issue links as comment to explain why we have to disable this by default.

Reply:
Thanks and added!